common/stub.go

```diff
@@ -0,0 +1,19 @@
+package common
+
+import (
+	"github.com/jaypipes/ghw/pkg/gpu"
+	"github.com/jaypipes/ghw/pkg/pci"
+)
+
+type StubHardware struct {
```

Comment:
1. If this stub is used only in `util_test.go`, could you move it to `util_test.go`? No need to create a separate go file.
2. The other thing is that you may not need this stub at all. Instead of having this stub, how about assigning the function directly, like this one?

```go
	// Test available GPU cards
	gpus := []*gpu.GraphicsCard{}
	for i := 0; i < 3; i++ {
		gpus = append(gpus, &gpu.GraphicsCard{
			DeviceInfo: &ghw.PCIDevice{
				Vendor: &pcidb.Vendor{
					Name: "--Nvidia Corp",
				},
			},
		})
	}

	GetGPU = func() ([]*gpu.GraphicsCard, error) {
		return gpus, nil
	}
```